test/test_etl.py

```diff
@@ -997,3 +997,15 @@ def test_deduplicate(self):
         tbl_expected = Table([["a", "b", "c"], [1, 2, 3], [1, 3, 2], [2, 3, 4]])
         tbl.deduplicate(["a", "b"])
         assert_matching_tables(tbl_expected, tbl)
+
+    def test_head(self):
+        tbl = Table([["a", "b"], [1, 2], [3, 4], [5, 6], [7, 8], [9, 10]])
+        tbl_expected = Table([["a", "b"], [1, 2], [3, 4]])
+        tbl.head(2)
```

Comment:
Looking at `tbl_expected`, this seems like it's returning three rows, not 2? Is the count zero-indexed? If so we should warn people in the docs.

Reply:
That's the `Table` constructor. The first array is the list of column names, then the subsequent arrays are the actual rows.

Reply:
Thanks for reviewing. I agree.

Reply:
Whoops 😂